tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -689,60 +667,6 @@ async function ensureCommerceMockLocalTestFixture(mockNamespace,
       mockNamespace,
       targetNamespace,
     withCentralApplicationConnectivity ? prepareFunction('central-app-gateway') : prepareFunction());
-  await waitForPodWithLabel('app.kubernetes.io/name', 'connector-service', 'kyma-integration');
-  await retryPromise(() => connectMockLocal(mockHost, targetNamespace), 10, 30000);
-  await retryPromise(() => registerAllApis(mockHost), 10, 30000);
-
-  if (withCentralApplicationConnectivity) {
-    await waitForDeployment('central-application-gateway', 'kyma-system');
-    await waitForDeployment('central-application-connectivity-validator', 'kyma-system');
```

Comment:
I think we should not remove these. They should be up and running there and we should wait for it

Reply:
We can wait for `central-application-gateway`, `connectivity-validator` stays only in compass scenario so we cannot really wait for it here